tests/integration/conftest.py

```diff
@@ -504,17 +638,19 @@ def _load_json_schema(path):
 
 
 @pytest.fixture
-def pre_registered(shell, request, yum_conf_exclude):
+def pre_registered(shell, request):
     """
     A fixture to install subscription manager and pre-register the system prior to the convert2rhel run.
-    For Oracle Linux we're using the _add_client_tools_repo_oracle to enable the client-tools repository
-    to install the subscription-manager package from.
-    We also exclude the rhn-client* packages for the same reason.
-    On Oracle Linux subscription-manger is obsolete and replaced by rhn-client* packages when installing subman.
+    We're using the client-tools-for-rhel-<version>-rpms repository to install the subscription-manager package from.
+        This does not apply to CentOS 8.5 since recently dependency conflicts caused inability to install the package
+        from client tools, repo. We install the package from the native repositories.
```

Comment:
Not applicable anymore. We set the releasever to 8.5 in the baseurl on CentOS 8.
https://github.com/oamg/convert2rhel/pull/946/files#diff-c87c5173e88bb0f60b00107f4586304f4869a976a4f998425efe3481c0db9f56R155
```suggestion
```

Reply:
![1hhv9m(1)](https://github.com/user-attachments/assets/9f50a3a5-18aa-421a-82df-aee4d72a10ab)